proof/refine/ARM/Schedule_R.thy

```diff
@@ -2012,9 +2034,18 @@ proof -
   note switchToThread_invs_no_cicd'[wp del]
   note switchToThread_lookupBitmapPriority_wp[wp]
   note assert_wp[wp del]
+  note if_split[split del]
+
+  (* if we only have one domain, we are in it *)
+  have one_domain_case:
+    "\<And>s. \<lbrakk> invs_no_cicd' s; numDomains \<le> 1 \<rbrakk> \<Longrightarrow> ksCurDomain s = 0"
+    by (simp add: all_invs_but_ct_idle_or_in_cur_domain'_def maxDomain_def)
+
+  (* NOTE: do *not* unfold numDomains in the rest of the proof,
+           it should work for any number *)
```

Comment:
Is this note still relevant? It seems to be covered by other, more centralised, comments, and it doesn't seem to relate to anything in this specific proof.

Reply:
- [ ] remove redundant "not unfold numDomains" comment in */Schedule_R